tox.ini

```diff
@@ -54,7 +54,10 @@ commands =
     {toxinidir}
 
 [testenv:flake8]
-deps = flake8
+deps =
+  flake8
+  flake8-docstrings
```

Comment:
I'd prefer pre-commit with pydocstyle, but this'll do for starters.

Reply:
@webknjaz Glad to hear about pre-commit, I wanted to enable it myself but I was not sure what's your take on it.